packages/integrations/mdx/src/plugins.ts

```diff
@@ -54,7 +54,7 @@ function getRemarkPlugins(mdxOptions: MdxOptions): PluggableList {
 		}
 	}
 
-	remarkPlugins = [...remarkPlugins, ...mdxOptions.remarkPlugins];
+	remarkPlugins = [...mdxOptions.remarkPlugins, ...remarkPlugins];
```

Comment:
Hmm, I'm not sure if this can cause issues or not. The image plugins being after should be fine, however maybe gfm and smartypants can cause issues? cc @bholmesdev

Reply:
Ah yeah sorry we did talk about this you're right. I rearrange them again so its gfm/smartyparts -> user defined -> image related -> syntax highlighters